<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QMainWindow" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>841</width>
      <height>811</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
border-radius:15px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>621</width>
       <height>641</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>fill out this field</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #d8d8d8;
</string>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>fill out this field</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(216, 216, 216);</string>
      </property>
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>fill out this field</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#d8d8d8;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>sign up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>410</y>
        <width>281</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semilight</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>create account</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>550</y>
        <width>321</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>17</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>have already an account?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>560</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Semibold</family>
        <pointsize>19</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>sign in</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>close</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(193, 0, 0);
border-radius:8px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Click to go back</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(170, 85, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../Pictures/icons8-return-100.png</normaloff>../Pictures/icons8-return-100.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>27</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="min">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>minimize</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 229, 0);
border-radius:8px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="max">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>10</y>
       <width>31</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>normal</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 0);
border-radius:8px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</normaloff>../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close</sender>
   <signal>clicked()</signal>
   <receiver>signup</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1094</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min</sender>
   <signal>clicked()</signal>
   <receiver>signup</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max</sender>
   <signal>clicked()</signal>
   <receiver>signup</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
